sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 # Release History
 
-## 12.21.0b1 (Unreleased)
+## 12.21.0b1 (2024-12-10)
 
 ### Features Added
+- Added support for service version 2025-05-05.
+- Added support for NFS shares to existing APIs. This includes adding `owner`, `group`, and `file_mode` options to
+`create_file`, `create_directory`, `set_http_headers` (File and Directory), `DirectoryProperties`, and `FileProperties`.
+- Added new `create_hard_link` method to `ShareFileClient` to create hard links to specified
+files and is only supported for the `NFS protocol`.
```

Comment:
meganit: I would only have `NFS` be in the backticks. Since technically [NFS ](https://github.com/Azure/azure-sdk-for-python/blob/86a8ccc863bfc8a665124a32df30a5693aa00a2f/sdk/storage/azure-storage-file-share/tests/test_nfs.py#L35)is the option, not `NFS protocol`. Typically we reserve backticks for valid symbols, methods, headers, etc.